imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -3,29 +3,45 @@ import { Meteor } from 'meteor/meteor'
 import { connect } from 'react-redux'
 import { createContainer } from 'meteor/react-meteor-data'
 import PropTypes from 'prop-types'
-import { Link, withRouter } from 'react-router-dom'
-import IconButton from 'material-ui/IconButton'
+import { withRouter, Link } from 'react-router-dom'
 import FontIcon from 'material-ui/FontIcon'
 import RaisedButton from 'material-ui/RaisedButton'
-import Cases, { collectionName, isClosed } from '../../api/cases'
+import Cases, { collectionName } from '../../api/cases'
 import { push } from 'react-router-redux'
 import RootAppBar from '../components/root-app-bar'
 import { storeBreadcrumb } from '../general-actions'
-
+import { CaseList, isClosed } from '../case-explorer/case-list'
 import {
-  unitIconsStyle,
-  moreIconColor
+  unitIconsStyle
 } from './case-explorer.mui-styles'
 
+const status = {
+  open: 0,
+  closed: 1,
+  assigned: 2
+}
+
 class CaseExplorer extends Component {
   constructor () {
     super(...arguments)
     this.state = {
       caseId: '',
       expandedUnits: [],
-      unitsDict: {}
+      unitsDict: {},
+      status: 0
```

Comment:
Change to a boolean as there are only two possible values (open/closed). The "assigned" mode shouldn't be in this PR and should have a value that is independent from the case openness status (all/assigned to me). 

It's better to rename "status" to something more meaningful like "showOpen". If it should have been an enum-like value, I would opt for a name like "filterType".